go/pkg/db/bulk_insert.go

```diff
@@ -0,0 +1,197 @@
+// Package db provides bulk database operations for the Unkey platform.
```

Comment:
Since sqlc doesn't support bulk inserts in mysql I've added this not super happy with it but it does the job. 

I just write the query for a single insert, generate params via sqlc and use them for the actual insert.